webapp/src/components/SingleStudy/InformationView.tsx

```diff
@@ -309,6 +316,23 @@ const InformationView = (props: PropTypes) => {
     }
   }, 2000, { leading: true, trailing: false });
 
+  const onExportFiltered = async (output: string, filter: StudyOutputDownloadDTO): Promise<void> => {
+    setOpenFilterModal(false);
+    if (study) {
+      try {
+        await downloadOutput(study.id, output, filter, false, true);
+        enqueueSnackbar(t('singlestudy:outputExportSuccess'), { variant: 'success' });
```

Comment:
as it is now a task, this is not a success, just a simple info message saying the export is ongoing